src/main/scala/cromwell/server/CromwellServer.scala

```diff
@@ -15,7 +17,18 @@ object CromwellServer extends WorkflowManagerSystem {
   val conf = ConfigFactory.load()
   val service = actorSystem.actorOf(CromwellApiServiceActor.props(workflowManagerActor, conf), "cromwell-service")
   val webserviceConf = conf.getConfig("webservice")
-  service.bindOrShutdown(interface = webserviceConf.getString("interface"), port = webserviceConf.getInt("port")) onSuccess {
-    case _ => actorSystem.log.info("Cromwell service started...")
+
+  def run(): Future[Any] = {
+    val interface = webserviceConf.getString("interface")
+    val port = webserviceConf.getInt("port")
+    val futureBind = service.bind(interface = interface, port = port)
+    futureBind andThen {
+      case Success(_) =>
+        actorSystem.log.info("Cromwell service started...")
+        actorSystem.awaitTermination()
+      case Failure(throwable) =>
+        actorSystem.log.error(throwable, s"Binding failed interface $interface port $port")
```

Comment:
maybe a synchronous `System.err.println` to make sure this message gets out at least once